sda-commons-server-spring-data-mongo/src/main/java/org/sdase/commons/server/spring/data/mongo/SpringDataMongoBundle.java

```diff
@@ -106,10 +106,7 @@ public class SpringDataMongoBundle<C extends Configuration> implements Configure
 
   private boolean morphiaCompatibilityEnabled = false;
 
-  /**
-   * Database as defined by the {@link MongoConfiguration#getConnectionString()} or {@link
-   * MongoConfiguration#getDatabase()}
-   */
+  /** Database as defined by the {@link MongoConfiguration#getConnectionString()} */
```

Comment:
[Sonar claims](https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&pullRequest=2949&id=SDA-SE_sda-dropwizard-commons&open=AYw1Ve5xyUtHArYJIgvz), that the `config` field above (sorry can't add comments to unchanged lines) can be a local variable now.